edge/bind/named.conf.local

```diff
@@ -0,0 +1,32 @@
+view "internal" {
```

Comment:
Internal view allows only the localhost and internal servers to query from corp.sangria.pi and sangria.pi and pi.